src/main/java/com/google/sps/seeder/AppSeeder.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.seeder;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+public class AppSeeder {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    public void seedDB() {
+        setEntityProperties("social", "connect with friends", "mockIcon.url", "com.facebook.ocra", "Messenger", 0, 4.6);
```

Comment:
Please seed the actual link to the mock icon url, which can be found here: https://github.com/googleinterns/play-web-step-2020/blob/master/src/main/java/com/google/sps/models/TopChartsCluster.java#L37